libavcodec/vvc/vvc_ps.c

```diff
@@ -1012,6 +1021,30 @@ static int sps_parse_vui(VVCSPS *sps, GetBitContext *gb, void *log_ctx)
     return 0;
 }
 
+static int sps_parse_extension(VVCSPS *sps, GetBitContext *gb, void *log_ctx)
+{
+    if (get_bits1(gb)) {  // sps_extension_present_flag
+        const unsigned int range_extension_flag = get_bits1(gb);
+        skip_bits(gb, 7);  // sps_extension_7bits
+        if (range_extension_flag) {
+            sps->extended_precision_flag = get_bits1(gb);
+            if (sps->transform_skip_enabled_flag) {
+                sps->ts_residual_coding_rice_present_in_sh_flag = get_bits1(gb);
+            }
+            sps->rrc_rice_extension_flag = get_bits1(gb);
+            sps->persistent_rice_adaptation_enabled_flag = get_bits1(gb);
+            sps->reverse_last_sig_coeff_enabled_flag = get_bits1(gb);
+        }
+    }
+
+    // Derived values
+    sps->log2_transform_range = sps->extended_precision_flag
```

Comment:
break ternary operator like [this]:(https://github.com/FFmpeg/FFmpeg/blob/c44fe101603dbe1e788079f4a798eeca13dbedcd/libavcodec/av1dec.c#L733)